profiles/schema.py

```diff
@@ -0,0 +1,76 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+        fields = ["vocabulary", "code", "translations"]
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+
+
+class DivisionOfInterestType(DjangoObjectType):
```

Comment:
do we actually need this type? since we are not really returning it via GraphQL at any point